openedx/features/course_experience/views/course_home.py

```diff
@@ -146,7 +156,9 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
             resume_course_url = None
 
         # Get the handouts
-        handouts_html = self._get_course_handouts(request, course)
+        handouts_html = ''
+        if user_access['is_enrolled'] or user_access['is_staff'] or allowed_public:
+            handouts_html = self._get_course_handouts(request, course)
```

Comment:
Showing the handouts for anonymous users is also out of scope of this task as far as I understand. In that case, you can move this to the first `if` clause above instead.

Reply:
👍